scripts/clear-all-fault-leds.sh

```diff
@@ -68,23 +86,37 @@ then
         GetSubTreePaths sias "/xyz/openbmc_project/inventory" 0 1 "xyz.openbmc_project.State.Decorator.OperationalStatus" \
         | sed  's/ /\n/g' | tail -n+3 | awk -F "\"" '{print $2}' | while read -r line
     do
-        echo "$line" | grep "powersupply\|dimm\|dcm\|fan" >/dev/null
+        echo "$line" | grep "powersupply\|dcm\|fan" >/dev/null
```

Comment:
so this still won't set functional back to true for PSs and fans?  I forget, why is that?

Reply:
we will ultimately try and fix the PS and fans. this fix is for DIMMs only